streamer/configuration.py

```diff
@@ -458,10 +470,13 @@ def _sortable_properties(self) -> Tuple:
     """Return a tuple of sortable properties.  Implemented by subclasses."""
     raise RuntimeError('_sortable_properties missing on RuntimeMapSubclass!')
 
-  def __eq__(self, other: Any) -> bool:
+  # NOTE: ignore[override] is needed to suppress the following the mypy
+  # error: "This violates the Liskov substitution principle,
+  # See https://mypy.readthedocs.io/en/stable/common_issues.html#incompatible-overrides"
+  def __eq__(self, other: 'RuntimeMap') -> bool: # type: ignore[override]
```

Comment:
Well, that `Any` must have a `_sortable_properties` method for it to be comparable with `self: RuntimeMap`, so that `Any` is a `RuntimeMap`.
These magical methods are used to compare objects of type `VideoResolution` & `AudioChannelLayout` which are `RuntimeMap`s.